frontend/package.json

```diff
@@ -1,21 +1,45 @@
 {
-  "name": "ecqm-dedupe-desktop-app",
+  "name": "deduplifhir-app",
   "version": "1.0.0",
-  "description": "",
+  "productName": "DedupliFHIR",
+  "description": "An Electron app that uses the DedupliFHIR CLI to provide data-linkage and AI deduplication for reported ACO data at scale.",
+  "author": "Natalia Luzuriaga",
+  "license": "CC0-1.0",
   "main": "main.js",
   "scripts": {
     "start": "electron .",
     "lint": "eslint ./**.js",
     "lint:fix": "eslint --quiet --fix \"**.js\"",
     "prettier": "prettier --check ./**.{js,css}",
     "prettier:fix": "prettier --write \"./**.{js,css}\"",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "pack": "electron-builder --dir",
+    "dist": "../set-up-python-env.sh && electron-builder"
   },
-  "author": "",
-  "license": "ISC",
+  "build": {
+    "appId": "com.example.app",
```

Comment:
Will this need to change in the future or does it not show up anywhere? We don't necessarily need to fill it in, but if it's used we should probably include something like `gov.cms.deduplifhir` for a namespace